themes/10up-block-theme/src/Blocks.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Gutenberg Blocks setup
+ *
+ * @package TenupBlockTheme
+ */
+
+namespace TenupBlockTheme;
+
+use TenupFramework\Module;
+use TenupFramework\ModuleInterface;
+use TenupBlockTheme\Traits\GetAssetInfo;
+
+/**
+ * Blocks module.
+ *
+ * @package TenupBlockTheme
+ */
+class Blocks implements ModuleInterface {
+
+	use Module;
```

Comment:
Stupid question: 

Why do we need to add `use Module` here? Isn't that already coming from the `implements ModuleInterface`?

Reply:
Not a stupid question! The `ModuleInterface` tells the class what methods it _needs_ to implement, the `Module` trait does the actual implementation.

The Interface/Trait approach do the same thing as the old Module abstract class used to, without the downside of _everything_ having to inherit a single class. This makes it more flexible for engineers to change those methods when they have things they actually want to do.